src/flownet/templates/THPRES.jinja2

```diff
@@ -2,6 +2,6 @@
 -- FROM TO VALUE
 THPRES
 {% for i in range(eqlnum1|length) -%}
- {{ eqlnum1[i] + 1 }} {{ eqlnum2[i] + 1 }} 1* /
+ {{ eqlnum1[i]}} {{ eqlnum2[i]}} 1* /
```

Comment:
Why is this not needed anymore? Does this not fail for the individual equil?

Reply:
The individual option will still go through this conversion
```
 eqlnum_combinations = list(
                {
                    (
                        self._eqlnum.loc[tube_a, "EQLNUM"],
                        self._eqlnum.loc[tube_b, "EQLNUM"],
                    )
                    for _, [tube_a, tube_b] in enumerate(eqlnum_combinations)
                    if self._eqlnum.loc[tube_b, "EQLNUM"]
                    > self._eqlnum.loc[tube_a, "EQLNUM"]
                }
            )
```
where you go from tube number to equilibrium region number (for individual this will be equal to +1)